docs/data-access/data-access.md

```diff
@@ -4,33 +4,44 @@ title: Applying for Access to Controlled Data
 sidebar_label: Applying for Access to Controlled Data
 ---
 
-The **Data Access Compliance Office (DACO)** handles approval for accessing controlled molecular data in the ARGO Data Platform.
+The **Data Access Compliance Office (DACO)** handles requests and approvals for accessing controlled molecular data in the ARGO Data Platform.
+
+## Guidelines for Applying for Access to Controlled Data
+
+While there is great potential of data sharing for scientific research, caution is required when sharing data about individuals participating in genomic research because genomic data, like other types of medical data, can contain personal and identifying information.As such, DACO applications will only be considered for individuals or organizations with qualified scientists researching public health objectives.
+
+As a prerequisite to applying for DACO access, pleae ensure that you:
+
+- are associated to a research organization or company
+- have a signing officer at your institution
```

Comment:
could also include what they call this on the application?
- have a signing officer at your institution (authorized institutional representative)